documentation/deliverables/individual_deliverables/open-source-group-kickoff.md

```diff
@@ -0,0 +1,148 @@
+
+### Communication tools onboarding   
+
+There are some basic communication tools that are required for a good onboarding experience for an open source contributor. Configuration and setting up an obboarding experinece for communications and contributing tools such as a Github and Slack are required. Specify the platforms, integration methods, user access levels, onboarding experience, and any technical considerations for making these tools fully operational for the community. 
+
+### Team meetings and open source events
+
+Detail the aspects of scheduling and conducting team scrum meetings. This can include information about the chosen video conferencing tools, meeting frequency, methods for the general public to join open meetings, and any requirements for ensuring effective remote communication during these meetings. 
+
+### Getting Started Guide Development  
+
+There is clear documentation and guidelines for the general public to get started and contribute. Documentation typically should include project overviews, installation instructions, usage guidelines, contribution procedures, and community engagement details, fostering a collaborative and informed open source community. 
+
+### Developer Tools Setup  
+
+Provide technical instructions for setting up developer tools and environments. This can encompass version control systems, code review tools, and any collaborative platforms for coding and testing.
+
+
+
+### Definition of done
+
+- [ ] Onboarding experience ready for the public this includes:
+    - [ ] onboarding to the selected chat tool, Slack
+    - [ ] onboarding to Github
+    - [ ] materials such as a deck or another format for getting started with the project
+    - [ ] onboarding guidance for developer tools and environments
+- [ ] The selected video conference tool, Zoom, is procured and ready for use for webinars, office hours, or other open source events. 
+
+### Proposed metrics for measuring goals/value/definition of done
+
+- [ ] Number of users onboarded to the open source community
+- [ ] Time to onboard to the open source community
+
+
+## Planning
+
+### Assumptions & dependencies
+<!-- Required -->
+
+What capabilities / milestones do we expect to be in place at the beginning of work
+on this milestone?
+
+- Procurement of the tools should be in place as the procurement process can delay delivery. Zoom, Slack, and Github are already licensed on the project. 
+
+Are there any notable capabilities / milestones we do NOT expect to be in place at the
+beginning of work on this milestone?
+
+
+
+### Open questions
+<!-- Optional -->
+
+- [ ] Is it important to have a public wiki available and ready in this deliverable?
+- [ ] Is it important to have a public API documentation ready for consumption in this deliverable? 
+
+### Not doing
+<!-- Optional -->
+
+The following work will *not* be completed as part of this milestone:
+- tools that are not stated as required (Google Groups, public wiki, etc) 
```

Comment:
I will include this in the security section but I believe we will need to get security approval to use Google Groups and maybe Zoom since they will be collecting email addresses of users. Slack and wiki are used currently so I assume they are OK but we may need to check on that with the Security team too